librz/core/disasm.c

```diff
@@ -348,6 +348,12 @@ static void ds_end_line_highlight(RDisasmState *ds);
 static bool line_highlighted(RDisasmState *ds);
 static int ds_print_shortcut(RDisasmState *ds, ut64 addr, int pos);
 
+// add for str issue
```

Comment:
Please don't use global variables - Rizin is a library, theoretically can have multiple contexts to work in, it would break the logic.

Reply:
Also please write more meaningful commit message and PR description, see our
- [Contribution Guide](https://github.com/rizinorg/rizin/blob/dev/CONTRIBUTING.md)
- [Developers Guide](https://github.com/rizinorg/rizin/blob/dev/DEVELOPERS.md)

Reply:
Also there is no need for these `// add for str issue` comments.

Reply:
OK. I will modify it.